.goreleaser.yml

```diff
@@ -0,0 +1,89 @@
+project_name: terraform-docs
+
+builds:
+  - main: .
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -w
+      - -X github.com/terraform-docs/terraform-docs/internal/version.version={{ .Version }}
+      - -X github.com/terraform-docs/terraform-docs/internal/version.commitHash={{ .ShortCommit }}
+      - -X github.com/terraform-docs/terraform-docs/internal/version.buildDate={{ .Date }}
+    goos:
+      - darwin
+      - linux
+      - freebsd
+      - windows
+    goarch:
+      - amd64
+      - arm
+    goarm:
+      - 7
+    ignore:
+      - goos: darwin
```

Comment:
Note that with the release the Mac M1 MacBook Pros, Darwin ARM is now something that should be supported. Right now they do support translating Intel to ARM, but having a truly native build would be useful.

Reply:
That's a good point. Based on your comments on brew formula, would it be enough if we build with `GOOS=darwin` and `GOARCH=arm64`?

Reply:
Actually it seems that wouldn't be possible, https://github.com/goreleaser/goreleaser/pull/1956 attempts to fix it but it has a hard dependency on go 1.16.